core/repo.go

```diff
@@ -84,6 +113,9 @@ type (
 		// the datastore with incomplete builds.
 		ListIncomplete(context.Context) ([]*Repository, error)
 
+		// ListRunningStatus returns a list of build / repository /stage information for builds that are incomplete.
+		ListRunningStatus(context.Context) ([]*RepoBuildStageStatus, error)
```

Comment:
since the response returns pending and running, maybe we should find a different name.  I would suggest we deprecated `ListIncomplete` in a future release since it doesn't add much value and it is undocumented.  Maybe for now we call it `ListIncompleteV2` or something like that?